packages/block-library/src/comment-template/index.php

```diff
@@ -30,14 +36,21 @@ function block_core_comment_template_render_comments( $comments, $block ) {
 		if ( ! empty( $children ) ) {
 			$inner_content  = block_core_comment_template_render_comments(
 				$children,
-				$block
+				$block,
+				$depth + 1
 			);
 			$block_content .= sprintf( '<ol>%1$s</ol>', $inner_content );
 		}
 
-		$content .= '<li>' . $block_content . '</li>';
+		$comment_classes = comment_class( '', $comment->comment_ID, $comment->comment_post_ID, false );
```

Comment:
The last parameter which is `false` here means that you do NOT want to `echo` the output but rather that you want to [return it from the function](https://developer.wordpress.org/reference/functions/comment_class/#parameters).

Reply:
Thanks for explaining! Let's add a comment in the code to that effect:

```suggestion
		// The `false` parameter at the end means that we do NOT want the function to `echo` the output but to return a string.
		// See https://developer.wordpress.org/reference/functions/comment_class/#parameters
		$comment_classes = comment_class( '', $comment->comment_ID, $comment->comment_post_ID, false );
```